### DISCUSSION TITLE
Extend `java-version` datasource

### DISCUSSION THREAD

**Discussion by viceice** [renovatebot/renovate#22688] [Suggest an Idea]
### Tell us more.

I found interesting metadata files[^2] on the `asdf-java` repo[^1].
We could enhance our `java-version` datasource with those versions and support them on containerbase.
These files are easy to parse and contain sha256 checksums to verify packages.

WDYT?

[^1]: https://github.com/halcyon/asdf-java
[^2]: https://github.com/halcyon/asdf-java/tree/master/data

**Comment by dhduvall** [comment]
Looks like they're derived from https://github.com/joschi/java-metadata/, which is probably even easier to use. Not sure I know enough to comment on this otherwise.
